contracts/cairo/upgrade.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# ANSI format
+GREEN='\e[32m'
+PURPLE='\033[1;34m'
+PINK='\033[1;35m'
+COLOR_RESET='\033[0m'
+
+if [ -f ./contracts/cairo/.env ]; then
+    echo "Sourcing .env file..."
+    source ./contracts/cairo/.env
+else
+    echo "Error: .env file not found!"
+    exit 1
+fi
+
+# Starkli implicitly utilizes these environment variables, so every time we use Starkli,
+# we avoid adding flags such as --account, --keystore, and --rpc.
+export STARKNET_ACCOUNT=$STARKNET_ACCOUNT
+export STARKNET_KEYSTORE=$STARKNET_KEYSTORE
+export STARKNET_RPC=$STARKNET_RPC
```

Comment:
README.md file says env variable is SN_RPC_URL, change here or in readme + scripts